content/en/docs/tasks/observability/metrics/tcp-metrics/index.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Collecting Metrics for TCP services
+description: This task shows you how to configure Istio to collect metrics for TCP services.
+weight: 20
+keywords: [telemetry,metrics,tcp]
+aliases:
+    - /docs/tasks/telemetry/tcp-metrics
```

Comment:
If this is a totally new file and there aren't any files being deleted (just looking at this PR), is there a need for the aliases?

Reply:
This is a new file for old tcp-metrics page. So, keep the aliases?

Reply:
Yes. That way old URL still comes here.